kernel/aster-nix/src/process/signal/sig_set.rs

```diff
@@ -3,31 +3,31 @@
 use super::{constants::MIN_STD_SIG_NUM, sig_num::SigNum};
 
 #[derive(Debug, Copy, Clone, Default, PartialEq, Eq)]
-pub struct SigMask {
```

Comment:
While I agree `SigSet` is, in general, a better name than `SigMask`. But I think we should still keep `SigMask` in the form of a type alias.

```rust
/// A signal mask.
pub type SigMask = SigSet;
```

The rationale is that most users of `SigSet` actually use the type to represent signal masks, instead of a set of signals. For example, the following code snippet, which uses `SigMask`,

```rust
        match mask_op {
            MaskOp::Block => {
                let mut new_sig_mask = SigMask::from(new_set);
```

is more readable than the version uses `SigSet`.

```rust
        match mask_op {
            MaskOp::Block => {
                let mut new_sig_mask = SigSet::from(new_set);
```